rasa/nlu/tokenizers/convert_tokenizer.py

```diff
@@ -30,16 +39,120 @@ class ConveRTTokenizer(WhitespaceTokenizer):
+                path to the directory containing the model files."""
+            )
+
+        if io.is_valid_remote_url(model_url):
+
+            if model_url == ORIGINAL_TF_HUB_MODULE_URL:
+                # Can't use the originally hosted URL
+                raise RasaException(
+                    f"""Parameter "model_url" of "ConveRTTokenizer" was
+                    set to "{model_url}" which does not contain the model any longer.
+                    You can either use a community hosted URL or if you have a
+                    local copy of the model, pass the path to the directory
+                    containing the model files."""
+                )
+
+            if model_url == RESTRICTED_ACCESS_URL:
+                # Can't use the URL that is reserved for tests only
+                raise RasaException(
+                    f"""Parameter "model_url" of "ConveRTTokenizer" was
+                    set to "{model_url}" which is strictly reserved for pytests of Rasa Open Source only.
+                    Due to licensing issues you are not allowed to use the model from this URL.
+                    You can either use a community hosted URL or if you have a
+                    local copy of the model, pass the path to the directory
+                    containing the model files."""
+                )
+
+        elif os.path.isfile(model_url):
+            # Definitely invalid since the specified path should be a directory
+            raise RasaException(
+                """Parameter "model_url" of "ConveRTTokenizer" was
+                set to the path of a file which is invalid. You
+                can either use a community hosted URL or if you have a
+                local copy of the model, pass the path to the directory
+                containing the model files."""
+            )
+
+        elif os.path.isdir(model_url):
+            # Looks like a local directory. Inspect the directory
+            # to see if model files exist.
+            self._validate_model_files_exist(model_url)
+            # Convert the path to an absolute one since
+            # TFHUB doesn't like relative paths
+            model_url = os.path.abspath(model_url)
+
+        else:
+            raise RasaException(
+                f"""{model_url} is neither a valid remote URL nor a local directory.
+                You can either use a community hosted URL or if you have a
+                local copy of the model, pass the path to
+                the directory containing the model files."""
+            )
+        return model_url
+
+    def __init__(
+        self, component_config: Dict[Text, Any] = None, ignore_exceptions: bool = False
+    ) -> None:
+        """Construct a new tokenizer using the WhitespaceTokenizer framework.
+
+        Args:
+            component_config: User configuration for the component
+            ignore_exceptions: Whether to ignore all exception checks. This
+            is strictly only for pytests.
```

Comment:
This seems misleading. It only disables exception checks for the model url, but not for anything else.